process-automation/latest/model/connectors/docusign.md

```diff
@@ -98,6 +98,57 @@ When a Process Automation process is instantiated this way, the following variab
 | envelopeId | string | Envelope ID of the document. |
 | documents | array | Documents related to the envelope and data related to them like uri, and id. |
 
+#### Configuration
+
+1. Log into the DocuSign eSignature console and click Settings on the top right.
+
+    ![Status Active]({% link process-automation/images/status-active.png %})
+
+2. Click the **ADD CONFIGURATION** dropdown list and select **Custom**.
+
+    ![Custom Configuration]({% link process-automation/images/custom-configuration.png %})
+
+3. Under Listener Settings select **Active Connection** from the Status dropdown list.
+
+4. Under Listener Settings enter your **URL to Publish** URL.
+
+    The pattern of the URL must follow: `{domain-name}/{application-name}/connector/{connector_name}/listener`.
+
+5. Under **Event settings > Trigger events > Envelope Events**, ensure the following are selected:
+
+    * **Envelope Sent**
+    * **Envelope Delivered**
+    * **Envelope Signed/Completed**
+    * **Envelope Declined**
+    * **Envelope Voided**
+
+6. Under **Event settings > Trigger events > Recipient Events**, ensure the following are selected:
+
+    * **Recipient Sent**
+    * **Recipient Auto Responded**
+    * **Recipient Delivered**
+    * **Recipient Signed/Completed**
+    * **Recipient Declined**
+    * **Recipient Authentication Failure**
+
+        ![Custom Configuration]({% link process-automation/images/settings.png %})
```

Comment:
Cannot really see what's in this picture, if it matters

Reply:
Ive removed them all